TestAssets/TestProjects/AppWithDependencyOnMalformedPackage/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "BadPackage": "1.0.0"
```

Comment:
I see this package checked in above, is there a way to generate a bad package so we can avoid checking it in? When things like tfms change, it can be tough to maintain these checked in nupkgs since the code to re-generate them isn't here.